README.md

```diff
@@ -116,6 +116,7 @@ A/B testing, canary release, blue-green deployment, limit rate, defense against
   - [CORS](docs/en/latest/plugins/cors.md) Enable CORS(Cross-origin resource sharing) for your API.
   - [URI Blocker](docs/en/latest/plugins/uri-blocker.md): Block client request by URI.
   - [Request Validator](docs/en/latest/plugins/request-validation.md)
+  - [CSRF](docs/en/latest/plugins/csrf.md) Based on the [`Double Submit Cookie`](https://en.wikipedia.org/wiki/Cross-site_request_forgery#Double_Submit_Cookie) way, protect your API from CSRF attacks.
```

Comment:
@Baoyuantop Hi, we also have a Chinese version, could you please add it too?

Reply:
Done